### ISSUE TITLE
Print summary of JITServer statistics at shutdown

### ISSUE THREAD

**Issue by mpirvu** [eclipse/openj9#9339]
The JIT prints a summary of compilation statistics at shutdown to stderr. This can be enabled by the `TR_PrintCompStats` env var. Example:
```
Number of compilations per level:
Level=1 numComp=1575
Level=2 numComp=3812
Level=3 numComp=1
Level=4 numComp=1
Level=5 numComp=1
NumJNIMethodsCompiled=192
NumMethodsFoundInSharedCache=2080
NumMethodsTakenFromSharedCache=2253
NumAotedMethods=556
NumberOfAotedMethodsThatWereRecompiled=633 (forced=0)
Classes loaded=9440
```
We need to enhance the output with stats about JITClient specific activity:
1) How many compilations were remote
2) How many messages were sent

This will give us some quick way of validating that the JDK is behaving as expected, without generating a verbose log.

**Comment by mpirvu** [comment]
@EmanElsaban Could you please look into this as your next issue? Thanks

**Comment by mpirvu** [comment]
More info on this issue:
The summary will be printed in `void TR::CompilationInfo::stopCompilationThreads()` when all the other summaries are printed.
Stats will be stored in `TR::CompilationInfo`around the point where `int32_t                _statsOptLevels[numHotnessLevels];` are defined
Stats about remote compilations could be incremented in `void TR::CompilationInfoPerThreadBase::logCompilationSuccess` at the point where `_statsOptLevels` are incremented. (_methodToBeCompiled->isRemoteCompReq() can act as a filter).

Routine
`JITServerHelpers::printJITServerMsgStats(J9JITConfig *jitConfig)` could print a total of messages, so that we don't have to add them manually.

**Comment by mpirvu** [comment]
Implemented in #9671.
